tests/optimizers/botorch_base_test.py

```diff
@@ -118,14 +127,6 @@ def test_acquisition_function_optimizer_factory_force_continuous():
 
     assert af_opt.func == optimize_acqf  # pylint: disable=no-member
 
-    af_opt = _acquisition_function_optimizer_factory(
-        discrete_space,
-        af_opt_kwargs={"raw_samples": 5_000},
-        torch_dtype=torch.float64,
-    )
-
-    assert af_opt.func == optimize_acqf  # pylint: disable=no-member
-
```

Comment:
Why do we need to delete this part of the test? It seems to me like this is still relevant and we would benefit from keeping it around.

Reply:
True. I think that it was just a mistake. Thanks for spotting this.